bundle_sync/main.go

```diff
@@ -116,8 +117,8 @@ func main() {
 		skus := make(map[string][]string)
 		current_skus := make(map[string][]string)
 		url := fmt.Sprintf("%s%s%s",
-			options.GetString("SUBS_HOST"),
-			options.GetString("SUB_API_BASE_PATH"),
+			options.GetString(config.Keys.SubsHost),
```

Comment:
can we have the same for the code above in the line 93?

Reply:
ah yes, somehow didn't notice that haha